meson.build

```diff
@@ -0,0 +1,228 @@
+project('freetype2', 'c',
+  license : ['FTL', 'GPL2'],
+  meson_version: '>= 0.47.0',
+  version: '22.1.16',
```

Comment:
Where this version is coming from?

Reply:
iirc that's the "libtool" version, whatever that means, we probably actually want 2.9.0 here (2.9.1? was this rebased?), see https://git.savannah.gnu.org/cgit/freetype/freetype2.git/tree/docs/VERSIONS.TXT#n56

Reply:
Please add a comment saying so. It will help a lot to anyone reading the wrap later.

Reply:
Now using the release version number eg 2.9.1